### DISCUSSION TITLE
[Spring] 컨텍스트 캐싱과 @MockBean

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#272] [1. 개발 관련 글 모음]
요즘 `@SpringBootTest`에 대해 파고 있는데, 생각보다 뭐가 많고 어렵더라구요...^-ㅠ
이 다음엔 테스트 격리에 대해 작성해보도록 하겠습니다...

[게시글 링크](https://amaran-th.github.io/Spring/[Spring]%20%EC%BB%A8%ED%85%8D%EC%8A%A4%ED%8A%B8%20%EC%BA%90%EC%8B%B1(Context%20Caching)/)

**Comment by 02ggang9** [comment]
글이 깔끔하고 이해하기 쉬웠어요~ 👍👍
